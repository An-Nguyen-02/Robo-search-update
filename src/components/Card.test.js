import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Card from './Card';

it('card snapshot',()=>{
	expect(shallow(<Card />)).toMatchSnapshot();
})