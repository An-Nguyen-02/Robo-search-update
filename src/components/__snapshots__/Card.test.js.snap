// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card snapshot 1`] = `
<div
  className="bg-light-blue dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robot_img"
    src="https://robohash.org/undefined?size=300x300"
  />
  <div>
    <h2>
       
       
    </h2>
    <p>
       
       
    </p>
  </div>
</div>
`;
